Makefile

```diff
@@ -371,7 +371,7 @@ vendor:
 
 .PHONY: test-vendor
 test-vendor: vendor
-	@diff=$$(git diff vendor/); \
+	@diff=$$(git diff go.mod go.sum); \
 	if [ -n "$$diff" ]; then \
 		echo "Please run 'make vendor' and commit the result:"; \
```

Comment:
Shoudn't we suggest some different command now? What mechanism would cause a diff in those two files?

Reply:
Thanks, new check looks good to me.